Cubical/Core/Prelude.agda

```diff
@@ -47,10 +47,15 @@ cong f p = λ i → f (p i)
 
 -- This is called compPath and not trans in order to eliminate
 -- confusion with transp
-compPath : x ≡ y → y ≡ z → x ≡ z
-compPath {x = x} p q i =
-  hcomp (λ j → \ { (i = i0) → x
-                 ; (i = i1) → q j }) (p i)
+
+compPath-filler : ∀ {x y z : A} → x ≡ y → y ≡ z → I → I → A
+compPath-filler {x = x} p q j i =
+  hfill (λ j → λ { (i = i0) → x
+                  ; (i = i1) → q j }) (inc (p i)) j
```

Comment:
Sounds like you need to reduce one space of indentation.

Reply:
Indeed. @bbentzen can you please fix?

Reply:
Sure! :)